index.js

```diff
@@ -45,7 +45,7 @@ function fastifyView (fastify, opts, next) {
     return
   }
 
-  const dotRender = type === 'dot' ? viewDot.call(fastify, preProcessDot(templatesDir, options)) : null
+  const dotRender = type === 'dot' ? viewDot.call(fastify, preProcessDot.call(fastify, templatesDir, options)) : null
```

Comment:
Do we need the `this` context in that function?

Reply:
This one here fails withouthe the `this` context
https://github.com/fastify/point-of-view/blob/e32d4ca52ed3c62215d74626b9478855b4c6f559/index.js#L225